pybloom_live/pybloom.py

```diff
@@ -291,6 +291,43 @@ def add(self, key):
         filter.add(key, skip_check=True)
         return False
 
+    def copy(self):
+        """Return a copy of this scalable bloom filter.
+        """
+        new_filter = ScalableBloomFilter(initial_capacity=self.initial_capacity,
+                                         error_rate=self.error_rate,
+                                         mode=self.SMALL_SET_GROWTH)
```

Comment:
That would be nice if you could help to change it for `BloomFilter`. It doesn't block us from merging your pull request if you prefer to leave it for now.